{
  withExpectedInternalFailure(DuplicateIdClass.Externalizer.class,"Should have thrown a CacheException reporting the duplicate id");
}
